bedrock/pocket/templates/pocket/includes/platform-footer.html

```diff
@@ -26,6 +26,8 @@
                   class="svg-social twitter"
                   xmlns="http://www.w3.org/2000/svg"
                   viewBox="0 0 28 28"
+                  width="28px"
```

Comment:
I added these because it's helpful to have HTML size defaults in case CSS does not load or is blocked by user

Reply:
smart!